.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,13 +13,14 @@
 #### Type of Improvement
 
 <!-- Choose one or more from the following:
-- bug fix
-- performance improvement
-- documentation
-- code refactoring
-- code cleanup
-- backend code generation
-- new feature/API
+- Feature (or new API)
```

Comment:
can we just take this off and tell people to select the right label? So they don't have to mess with the PR template AND apply the label?

Reply:
You must have write access to set the label, so unfortunately no.

Reply:
We could make a workflow that auto labels based on the body of the PR though, but the usefulness vs. amount of work relative to how easy it is to label a PR makes me feel it isn't worth it... yet at least.